nss-day-cohort-41/tabloidmvc-the-gefilte-fish
# Add and Remove tags from post

# Description
- added ability to add and remove tags from posts
- added tag manager page
- edited post details page to include tags for that post

- tickets: [#17](https://github.com/nss-day-cohort-41/tabloidmvc-the-gefilte-fish/issues/17) & [#18](https://github.com/nss-day-cohort-41/tabloidmvc-the-gefilte-fish/issues/18)

## Type of change
# Testing Instructions
- clone repository
- navigate to root folder (of repo)
- run included sql scripts to create and populate database
- dotnet run
- navigate to the port that your system is running the build on in Google Chrome
- login 'admin@example.com'
- click Posts button on top right corner of screen
- user will be taken to post Index
- post details is an eye icon on the right side of the page
- user will be taken to post details
- existing post tags are display above post content
- tag icon just below the post title, to the right of the pencil and trash icons is the tag management button
- each tag will be presented as a checkbox option with current tags pre-checked
- check or uncheck tags
- click save 
- user will be redirected back to post details with updated tag selection

# Checklist:
